lib/workers/repository/init/index.ts

```diff
@@ -50,6 +51,7 @@ export async function initRepo(
   config = await initApis(config);
   await initializeCaches(config as WorkerPlatformConfig);
   config = await getRepoConfig(config);
+  setRepositoryLogLevelRemaps(config.logLevelRemap);
```

Comment:
How do you suggest to handle it?

Reply:
I'm not sure, maybe add something to the docs that you shouldn't remap errors or warnings because it can cause unexpected failures in future?

@rarkins WDYT?

Reply:
I thought that people remapping warnings would be one of the top use cases. I think we need to document to "use with caution" whenever mapping warn or error lower

Reply:
Two ideas:
- Introduce `allowDowngradeRemap` option, making only "upgrading" remaps by default
- Introduce `remapFrom: string | string[]` option, to limit the scope of possibly undesired remaps

Reply:
I'm not sure we need, after all we still keep the original level in the log, right?

Reply:
Okay